### PR TITLE
Show custom tank name

### PR DESCRIPTION
**Pull request by DanielMcInnes**
Fix bug in TanksTab. We were enabling the ExpandedTanksView when there was more than 1 tank of a specific type. We should only enable this when there are 'merged' tanks displayed in the delegate.

### CODE REVIEW

**Comment by DanielMcInnes** [comment]
Matthijs asked us to remove the ExpandedTanksView. I think he thought it was useless because we were enabling it when we shouldn't have. If you have 'merged' tanks it is more useful, you can see individual tank capacities / custom names etc.

**Comment by DanielMcInnes** [comment]
Screenshot from 2024-08-19 16-53-42
ExpandedTanksView is now only enabled for 'merged' tanks such as the Fresh Water tanks show here.

File: components/ExpandedTanksView.qml [hunk 1]
```diff
@@ -47,7 +47,7 @@ Rectangle {
 				width: Gauges.width(groupedSubgaugesRepeater.count, Theme.geometry_levelsPage_max_tank_count, root.width)
 				height: Theme.geometry_levelsPage_panel_expanded_height
 
-				header.text: model.device.name || root._tankProperties.name
+				header.text: model.device.customName || model.device.name || root._tankProperties.name
```
Comment (blammit) [inline components/ExpandedTanksView.qml:50]: This won't be needed if the tank name is fixed in the other PR.

File: pages/TanksTab.qml [hunk 2]
```diff
@@ -93,7 +93,7 @@ ListView {
 
 					animationEnabled: root.animationEnabled
 					tankType: tankTypeDelegate.tankType
-					header.text: mergeTanks ? tankProperties.name : model.device.name || tankProperties.name
+					header.text: mergeTanks ? tankProperties.name : model.device.customName || model.device.name || tankProperties.name
```
Comment (blammit) [inline pages/TanksTab.qml:96]: Same here

File: pages/TanksTab.qml [hunk 3]
```diff
@@ -108,7 +108,7 @@ ListView {
 
 					PressArea {
 						anchors.fill: parent
-						enabled: tankTypeDelegate.tankModel.count > 1
+						enabled:tankTypeDelegate.mergeTanks
```
Comment (blammit) [inline pages/TanksTab.qml:111]: Add space after `enabled:`?

**Review by blammit — APPROVED** [review]

**Pull request merged**